<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaHistoricoMedico</class>
 <widget class="QWidget" name="TelaHistoricoMedico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Historico MÃ©dico</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNome">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome_do_Medico</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVoltar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>441</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPaciente">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumero">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAbrir">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Paciente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCRM">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CRM</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
